.env.development

```diff
@@ -0,0 +1 @@
+REACT_APP_EXERCISM_HOST=http://lvh.me:3000
```

Comment:
I recommend `.gitignore` this file.

Reply:
I've gone a slightly different route after reading through [the docs](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env). I now have a single `.env` file. Locally, I can then have a `.env.development.local` file that overrides this. This works nicely!